docs/book/intro.md

```diff
@@ -1,6 +1,8 @@
+<!-- markdownlint-disable -->
 <noscript><meta http-equiv="refresh" content="0; url=/laminas-mvc-middleware/v2/intro/"></noscript>
```

Comment:
This should be deleted at all. Use redirects:

https://github.com/laminas/laminas-mvc-middleware/blob/db543d3489b8a83e094c29c02c903c23d4acacbd/mkdocs.yml#L25-L29

Reply:
Thanks @froschdesign - I can never remember what I'm supposed to do with these meta redirects. I'll send in a patch for the next release 👍